Sources/EmbraceCore/Capture/Network/URLSessionCaptureService.swift

```diff
@@ -116,6 +122,32 @@ struct URLSessionInitWithDelegateSwizzler: URLSessionSwizzler {
             }
         }
     }
+
+    // list of third party URLSessionDelegate implementations that we don't support
+    // due to issues / crashes out of our control
+    private let unsupportedDelegates: [String] = [
+
+        // This type belongs to an internal library used by Firebase which
+        // incorrectly assumes the type of the URLSession delegate, resulting
+        // in it calling a method that is not implemented by our proxy.
+        //
+        // We can't solve this on our side in a clean way so we'll just not
+        // capture any requests from this library until the issue is solved
+        // on their side.
+        //
+        // Library: https://github.com/google/gtm-session-fetcher/
+        // Issue: https://github.com/google/gtm-session-fetcher/issues/190
+        "GTMSessionFetcher"
+    ]
+
+    func isDelegateSupported(_ delegate: URLSessionDelegate?) -> Bool {
```

Comment:
Only concern is that if someone is using the workaround in https://github.com/google/gtm-session-fetcher/issues/190 we will still not be able to support these requests.

An alternative solution would be that we provide the workaround and ask people running into this issue to explicitly make this call before they start Embrace? We'd provide the snipper in some `ThirdPartySupport` target where they could call [`GTMSessionFetcherService. swizzleDelegateDispatcherForFetcher()`](https://github.com/google/gtm-session-fetcher/issues/190#issuecomment-607398226)?

Reply:
People have reported that the workaround doesn't work, that's why we had to do this in the first place.